### ISSUE TITLE
Mutation support

### ISSUE THREAD

**Issue by avwie** [sheerun/graphqlviz#25]
I notice there are no mutations in the visualisation. Is that on purpose?

**Comment by NathanRSmith** [comment]
Yes, this viz is focused on the Query aspects of GraphQL. If you have proposals for how mutations could be displayed, certainly open to hearing more about what might make this tool more useful.
